### ISSUE TITLE
""" -> command not found

### ISSUE THREAD

**Issue by asmeurer** [xonsh/xonsh#2436]
```
$ """
xonsh: subprocess mode: command not found:

```

It should start a docstring, like in Python.

**Comment by scopatz** [comment]
This produces the following traceback for me:

```
scopatz@artemis ~ $ """
xonsh: To log full traceback to a file set: $XONSH_TRACEBACK_LOGFILE = <filename>
Traceback (most recent call last):
  File "/home/scopatz/.local/lib/python3.6/site-packages/xonsh/__amalgam__.py", line 19854, in _parse_ctx_free
    debug_level=(self.debug_level > 2))
  File "/home/scopatz/.local/lib/python3.6/site-packages/xonsh/parsers/base.py", line 348, in parse
    tree = self.parser.parse(input=s, lexer=self.lexer, debug=debug_level)
  File "/home/scopatz/miniconda/lib/python3.6/site-packages/ply/yacc.py", line 331, in parse
    return self.parseopt_notrack(input, lexer, debug, tracking, tokenfunc)
  File "/home/scopatz/miniconda/lib/python3.6/site-packages/ply/yacc.py", line 1199, in parseopt_notrack
    tok = call_errorfunc(self.errorfunc, errtoken, self)
  File "/home/scopatz/miniconda/lib/python3.6/site-packages/ply/yacc.py", line 193, in call_errorfunc
    r = errorfunc(token)
  File "/home/scopatz/.local/lib/python3.6/site-packages/xonsh/parsers/base.py", line 2723, in p_error
    column=p.lexpos))
  File "/home/scopatz/.local/lib/python3.6/site-packages/xonsh/parsers/base.py", line 481, in _parse_error
    raise err
  File "<string>", line None
SyntaxError: <xonsh-code>:1:0: EOF in multi-line string
"""
^

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/scopatz/.local/lib/python3.6/site-packages/xonsh/ptk/shell.py", line 137, in _push
    locs=None)
  File "/home/scopatz/.local/lib/python3.6/site-packages/xonsh/__amalgam__.py", line 19798, in compile
    transform=transform)
  File "/home/scopatz/.local/lib/python3.6/site-packages/xonsh/__amalgam__.py", line 19767, in parse
    tree, input = self._parse_ctx_free(input, mode=mode, filename=filename)
  File "/home/scopatz/.local/lib/python3.6/site-packages/xonsh/__amalgam__.py", line 19872, in _parse_ctx_free
    line, nlogical, idx = get_logical_line(lines, idx)
  File "/home/scopatz/.local/lib/python3.6/site-packages/xonsh/__amalgam__.py", line 3909, in get_logical_line
    line = line + '\n' + lines[idx]
IndexError: list index out of range
```

**Comment by asmeurer** [comment]
I also get that on master. Before I was on the relesae.

**Comment by melund** [comment]
Fixed by #2441. @asmeurer  Feel free to reopen if it doesn't fix it for you.
